### PR TITLE
config: cut over from 4.5 to 4.6 for master

### PR DESCRIPTION
**Pull request by droslean**
/hold

generated with:
```sh
#!/bin/bash

set -o errexit
set -o nounset
set -o pipefail

sed -i 's/current-release=4.5/current-release=4.6/g' ci-operator/jobs/infra-periodics.yaml
sed -i 's/future-release=4.6/future-release=4.7/g' ci-operator/jobs/infra-periodics.yaml
git add ci-operator/jobs/infra-periodics.yaml
git commit --signoff -m "change release versions in infra periodics"

sed -i 's/base-4-5/base-4-6/g' ci-operator/config/openshift/images/openshift-images-master.yaml
git add ci-operator/config/openshift/images/openshift-images-master.yaml
git commit --signoff -m "change base release version in openshift images ci-operator configuration" 

sed -i -e 's/,latest//' -e 's/4.6.0/4.6.0,latest/' -e '/mapping_origin_4_6/ a \	BASE=quay.io/openshift/origin- VERSION=4.7 TAG=4.7,4.7.0 ../../../hack/mirror-file > mapping_origin_4_7' core-services/image-mirroring/openshift/Makefile
make -C core-services/image-mirroring/openshift image-mirror-files
git add core-services/image-mirroring/openshift
git commit --signoff -m "update mirror imaging for 4.6"


config-brancher --config-dir ci-operator/config --current-release=4.5 --bump-release=4.6 --future-release=4.6 --future-release=4.7 --confirm
git add ci-operator/config
git commit --signoff -m "config-brancher --config-dir ci-operator/config --current-release=4.5 --bump-release=4.6 --future-release=4.6 --future-release=4.7 --confirm"

determinize-ci-operator --config-dir ci-operator/config --confirm
git add ci-operator/config
git commit --signoff -m "determinize-ci-operator --config-dir /ci-operator/config --confirm"

ci-operator-prowgen --from-dir ci-operator/config --to-dir ci-operator/jobs
git add ci-operator/jobs
git commit --signoff -m "ci-operator-prowgen --from-dir ci-operator/config --to-dir ci-operator/jobs"

sanitize-prow-jobs --prow-jobs-dir ci-operator/jobs --config-path core-services/sanitize-prow-jobs/_config.yaml
git add ci-operator/jobs
git commit --signoff -m "sanitize-prow-jobs --prow-jobs-dir ci-operator/jobs --config-path core-services/sanitize-prow-jobs/_config.yaml"
```

/cc @openshift/openshift-team-developer-productivity-test-platform @stevekuznetsov

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
How is this differen from https://github.com/openshift/release/pull/8499?

**Comment by droslean** [comment]
Can't generate the job correclty until https://github.com/openshift/ci-tools/pull/725

File: ci-operator/config/kube-reporting/ghostunnel/kube-reporting-ghostunnel-release-4.7.yaml [hunk 1]
```diff
@@ -27,6 +27,6 @@ tag_specification:
   name: "4.7"
   namespace: ocp
 zz_generated_metadata:
-  branch: release-4.7
+  branch: master
```
Comment (droslean) [inline ci-operator/config/kube-reporting/ghostunnel/kube-reporting-ghostunnel-release-4.7.yaml:30]: @stevekuznetsov Is this a bug in `config-brancher`?

Comment (droslean) [reply ci-operator/config/kube-reporting/ghostunnel/kube-reporting-ghostunnel-release-4.7.yaml]: I added a step to `determinize-ci-operator`

**Pull request merged**